### ISSUE TITLE
Breaks hugo and gulp livereload

### ISSUE THREAD

**Issue by shreyasminocha** [typicode/hotel#203]
I've tried hotel with two livereload servers:

* hugo
* gulp

Unfortunately, accessing these through the `.dev` url breaks the livereload mechanism. Is there something I am missing?

I am willing to provide you with any information that will help you diagnose this(hotel configuration files etc).

**Comment by typicode** [comment]
Hi @shreyasminocha,

Are you using `https`?
In your browser network tab, what's the URL failing?

**Comment by shreyasminocha** [comment]
No, I'm not using `https`. It seems to be `ws://whatever.dev:port/livereload`
![screen shot 2017-07-08 at 18 12 38](https://user-images.githubusercontent.com/11537232/27985468-ff021930-6408-11e7-8daf-c58efa30b9f5.JPEG)

**Comment by typicode** [comment]
Hmm, I don't really know. Livereload support has been improved in v0.5.12 https://github.com/typicode/hotel/blob/master/CHANGELOG.md#0512 and it worked with `ember-cli` and `livereload`.

I'll probably have to retry with the current `hotel` version and `gulp`.

The current behaviour is that if you have a port after `.dev`, it should ignore the domain part and just use `port`. In other words, `ws://whatever.dev:port/livereload` should be the same as `ws://localhost:port/livereload`

**Comment by shreyasminocha** [comment]
@typicode I am extremely sorry, seems to be an issue at my end. `gulp` doesn't work even without `hotel`. I should have checked this earlier. I tried the same thing with hugo as well, in which case the issue persists(as shown in the screebshot above)

**Comment by typicode** [comment]
No worries, glad it seems to be solved with `gulp` :)
I'll try with `hugo` then.
